pages/3_detail.py

```diff
@@ -0,0 +1,82 @@
+import streamlit as st
+import plotly.graph_objs as go
+import pandas as pd
+from util.normalize_entries import normalize_entries
+from util.plot_graph import plot_graph
+from db.entries import get_entry_by_name
+
+if "name" not in st.query_params:
+    st.query_params["name"] = []
+
+name = st.query_params["name"]
+
+get_entries = get_entry_by_name(name)
+entries = normalize_entries(get_entries, ref_electrode="SHE", c_ref=None, ion=None, ref_scan_rate=None)
```

Comment:
```suggestion
all_entries = get_all_entries()
entry = entries[name]

entries = normalize_entries([entry], ref_electrode="SHE", c_ref=None, ion=None, ref_scan_rate=None)
```
Für den Zugriff auf ein spezifisches Element können wir die CVCollection wie ein dict nutzen.

Und wegen normalize_entries, entweder könnten wir die methode mit einer neuen Liste aufrufen (so wie in der suggestion), oder, was ich shcöner fände, wir refactoren `normalize_entries` in die Funktion `normalize_entry` die dann in `normalize_entries` aufgerufen wird und mit einem Eintrag arbeiten kann